collectors/cgroups.plugin/multi_metadata.yaml

```diff
@@ -1040,5 +1052,7 @@ modules:
         - proxmox
         - container
     overview:
+      <<: *overview
       data_collection:
+        <<: *data_collection
         metrics_description: "Monitor Proxmox for performance, resource usage, and health status."
```

Comment:
This collector doesn't monitor Proxmox for performance, resource usage, and health status

Reply:
The same goes for every other _virtual_ integration. I am not sure it matters, if does - should be changed.

Reply:
I think this is a @sashwathn @shyamvalsan question, I just added these from our meeting, I agree that my wording is a bit off, as it doesn't make sense for proxmox and the likes, but can't think of a better one at the top of my head / time was pressing

Reply:
cgroup.plugin monitors only cgroups, so it is always containers or/and VMs.

Reply:
Yeah agree, either we got to change the wording but I get this should not be in here at all.. Hence I asked the guys to give a bit of guidance on this